package.json

```diff
@@ -38,27 +38,27 @@
   },
   "devDependencies": {
     "7zip-bin": "^2.0.4",
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.3.2",
+    "babel-core": "^6.24.1",
+    "babel-eslint": "^7.2.3",
+    "babel-loader": "^7.0.0",
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-preset-react": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
     "chai": "^3.5.0",
-    "cross-env": "^3.1.4",
+    "cross-env": "^4.0.0",
     "devtron": "^1.4.0",
-    "electron": "1.6.2",
-    "electron-builder": "^14.5.3",
-    "electron-builder-squirrel-windows": "^15.0.0",
+    "electron": "1.6.6",
```

Comment:
1.6.8 was still beta when I created this PR. And 1.6.7 has important changes for Windows. 

- Desktop notification for Windows 7
- Per-monitor DPI awareness for Windows 8.1/10

So I want to take 1.6.6 for now.